### PR TITLE
Multi modal RAG benchmark

### PR DESCRIPTION
**Pull request by rlancemartin**
* Example notebooks for eval of multi-modal RAG w/ mm-embd and mv-retriever vs baseline top-k RAG
* To Do: create public benchmark

### CODE REVIEW

File: docs/source/notebooks/retrieval/multi_modal_benchmarking/DDOG_Q3_earnings_deck.pdf
Comment (hinthornw) [inline docs/source/notebooks/retrieval/multi_modal_benchmarking/DDOG_Q3_earnings_deck.pdf:1]: Ideally we don't check these into the repo and either use LFS or GCS

Comment (rlancemartin) [reply docs/source/notebooks/retrieval/multi_modal_benchmarking/DDOG_Q3_earnings_deck.pdf:1]: Ya, I'm removing now that we are creating the public dataset.

**Comment by hinthornw** [comment]
Fixed up the paths and notebook

**Comment by rlancemartin** [comment]
> Fixed up the paths and notebook

Nice. I will further clean both both notebook and add comments / context.

**Review by hinthornw — APPROVED** [review]

**Pull request merged**